src/MoreMenu.js

```diff
@@ -675,19 +675,43 @@ $(() => {
         return html;
     }
 
+    /**
+     * The #p-cactions element is always in the DOM, so we can use it
+     * to create our own menu without worrying about HTML or class changes.
+     *
+     * @param {string} parentKey
+     * @param {string} html of list items
+     * @throws {Error} if the element has been unexpectedly removed from the DOM
+     *  so that appropriate action can be taken.
+     * @return {string}
+     */
+    function makeMenuHtml(parentKey, html) {
```

Comment:
I would perhaps move all the contents of this function to `drawMenuVector()` (and possibly rename it to `buildMenuVector()`). It doesn't make much sense as a separate method, especially as this is specific to vector.

Reply:
Sorry I've been wrapped up in some other stuff. Feel free to revise as you wish. I think naming this function so that it is Vector specific sounds like a good idea.

Reply:
Done!